src/Pact/Types/Typecheck.hs

```diff
@@ -62,17 +65,25 @@ data CheckerException = CheckerException Info String deriving (Eq,Ord)
 instance Exception CheckerException
 instance Show CheckerException where show (CheckerException i s) = renderInfo i ++ ": " ++ s
 
+data Schema = Schema
+  { _utName :: TypeName
```

Comment:
```suggestion
  { _schName :: TypeName
```
etc ?

Reply:
Done. I also changed `ModSpec`'s accessor to reflect the same thinking.